specs/api.kuflow.com/v2022-10-08/openapi.yaml

```diff
@@ -61,6 +61,12 @@ tags:
       description: Find out more about our principal model.
       url: "https://docs.kuflow.com/fine-grained-concepts/users-groups-roles"
     x-displayName: Principal
+  - name: tenantUser
+    description: Operations about tenant users.
+    externalDocs:
+      description: Find out more about our principal model.
+      url: "https://kuflow.com"
```

Comment:
I think is better the doc url, this is for devs

Reply:
:+1: